app/assets/stylesheets/bento.scss

```diff
@@ -397,3 +397,6 @@ $z-depth-way-front:     1000;
   font-weight: $fw-bold;
 }
 
+.wrap-result>.result-pubinfo {
```

Comment:
We should probably create a debug mode that re-enables this and maybe some other stuff. But this is fine for now.